### DISCUSSION TITLE
Farbfeld

### DISCUSSION THREAD

**Discussion by Kresimir235** [ImageMagick/ImageMagick#2664] [Development] [answered]
Are there any plans to support farbfeld format? 

https://tools.suckless.org/farbfeld/

It is a very simple, open image format, somewhat similar to TARGA, but currently the support for it is very limited. Including it in ImageMagick would go a long way towards its acceptance.

**Comment by dlemstra** [comment, marked as answer]
We just published a new release of ImageMagick that includes support for this format.

**Reply by Kresimir235** [reply]
That's fantastic news!

**Reply by dlemstra** [reply]
To be clear this was only added to ImageMagick 7. We will not be adding new formats to ImageMagick 6.

**Reply by Kresimir235** [reply]
I figured as much. I just updated to 7 (Arch Linux repos already have it). Seems to work well. Thank you!

**Comment by blue-j** [comment]
I'm having a hard time finding any sample farbfeld.ff images.  Can someone please point me to some samples?  They don't seem to be in the distro or on the site linked.  - J

**Reply by Kresimir235** [reply]
Farbfeld sample images: https://telparia.com/fileFormatSamples/image/farbfeld/

**Comment by urban-warrior** [comment]
You can always create one:

`convert logo: logo.ff`
